Llk/Lexer.php

```diff
@@ -285,6 +288,16 @@ protected function matchLexeme($lexeme, $regex, $offset)
             return null;
         }
 
+        if (false === $preg) {
+            throw new InternalError(
```

Comment:
Use `Compiler\Exception\InternalError` instead of `InternalError`, in order to be consistent with the rest of the code. Thanks!